assets/Scripts/Managers/LoggingManager.cs

```diff
@@ -155,6 +162,46 @@ public class LoggingManager : MonoBehaviour {
 		date = System.DateTime.Now.ToString("yyyy-MM-dd");
 		time = System.DateTime.Now.ToString("HH:mm:ss:ffff");
 
+	Dictionary <string, List<string>> logs = new Dictionary<string, List<string>>() //create a new dictionary
```

Comment:
You only need to construct the dictionary once. I suggest you move it to the `Awake()` function. If you need to clear it, you can move it to a separate function which you call, when clearing is needed.

Reply:
I made some changes by doing what you suggested. So, now the dictionary is moved into the `awak()` function and constructed only once and it cleared with `resetLogs` function 🌖.
I learnt new best practice... thank you 😎